spec.bs

```diff
@@ -1433,11 +1439,16 @@ an [=interest group=] |ig|, a [=boolean=] |isComponentAuction| and a [=boolean=]
   The <dfn method for="InterestGroupReportingScriptRunnerGlobalScope">sendReportTo(|url|)</dfn>
   method steps are:
 
-  1. [=exception/Throw=] a {{TypeError}} and clears report url if any of the following conditions hold:
-    * The API is called more than once;
-    * |url|'s [=url/scheme=] is not "`https`".
-  1. If called in reportWin(), set reportWinURL to url.
-  1. If called in reportResult(), set reportResultURL to url.
+  1. If [=this=]'s [=relevant global object=]'s
+    [=InterestGroupReportingScriptRunnerGlobalScope/report url=] is not null:
+    1. Set [=this=]'s [=relevant global object=]'s

```

Comment:
I thought we allowed resetting the reporting URL?

Reply:
sendReportTo() doesn't allow being called more than once: https://crsrc.org/c/content/services/auction_worklet/report_bindings.cc;drc=75d0df9820c7253c31292acd090ec09a727f8dca;l=63.
We allowed resetting the reporting URL for forDebuggingOnly ones.